<ui version="4.0" >
 <class>ImportSpektraDialog_base</class>
 <widget class="QDialog" name="ImportSpektraDialog_base" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Import Spektra test files</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gex.qrc">
    <normaloff>:/gex/icons/gex_application_48x48.png</normaloff>:/gex/icons/gex_application_48x48.png</iconset>
  </property>
  <property name="autoFillBackground" >
   <bool>true</bool>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" name="hBoxLayoutFileName" >
     <item>
      <widget class="QLabel" name="labelFileName" >
       <property name="text" >
        <string>SPEKTRA file name:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSelect" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>Select or Drag over the STDF file to dump</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="C:/Work/Dev_QT4/Galaxy_Products/gex_product/gex/sources/gex.qrc" >
         <normaloff>:/gex/icons/file_open.png</normaloff>:/gex/icons/file_open.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonProperties" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>Show STDF file properties</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="C:/Work/Dev_QT4/Galaxy_Products/gex_product/gex/sources/gex.qrc" >
         <normaloff>:/gex/icons/properties.png</normaloff>:/gex/icons/properties.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="editFileName" >
     <property name="acceptDrops" >
      <bool>false</bool>
     </property>
     <property name="toolTip" >
      <string>STDF file to Dump...</string>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hBoxLayoutOutputFormat" >
     <item>
      <widget class="QLabel" name="labelOutputFormat" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
       <property name="text" >
        <string>Output format:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboOutputFormat" >
       <item>
        <property name="text" >
         <string>ASCII</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>STDF</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupRecords" >
     <property name="title" >
      <string>Records to Dump</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <item row="0" column="0" >
       <layout class="QVBoxLayout" name="hBoxLayoutCheckBoxRecords" >
        <item>
         <widget class="QCheckBox" name="checkLabel" >
          <property name="toolTip" >
           <string>FAR: File Attributes Record</string>
          </property>
          <property name="text" >
           <string>Label</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkTestItem" >
          <property name="toolTip" >
           <string>ATR: Audit Trail Record</string>
          </property>
          <property name="text" >
           <string>TestItem</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkWaferID" >
          <property name="toolTip" >
           <string>MIR: Master Information Record</string>
          </property>
          <property name="text" >
           <string>WaferID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkDeviceID" >
          <property name="toolTip" >
           <string>MRR: Master Results Record</string>
          </property>
          <property name="text" >
           <string>DeviceID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkTestData" >
          <property name="toolTip" >
           <string>PCR: Part Count Record</string>
          </property>
          <property name="text" >
           <string>TestData</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" name="hBoxLayoutButtonRecords" >
        <item>
         <widget class="QPushButton" name="buttonSetAllRecords" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Set all</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonClearAllRecords" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Clear all</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupOptions" >
     <property name="title" >
      <string>Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <layout class="QHBoxLayout" name="hBoxLayoutFieldFilter" >
        <item>
         <widget class="QLabel" name="labelFieldFilter" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="text" >
           <string>Dump style:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboFieldFilter" >
          <item>
           <property name="text" >
            <string>Detailed (list all fields in records)</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Short (list record names only, no data fields)</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" name="hBoxLayoutSizeLimit" >
        <item>
         <widget class="QLabel" name="labelSizeLimit" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>80</width>
            <height>20</height>
           </size>
          </property>
          <property name="text" >
           <string>Output size:</string>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboSizeLimit" >
          <property name="toolTip" >
           <string>Maximum size of the Dump</string>
          </property>
          <item>
           <property name="text" >
            <string>No limit</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>500 kb maximum</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>1 Mb maximum</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>2 Mb maximum</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>3 Mb maximum</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>4 Mb maximum</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>5 Mb maximum</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>10 Mb maximum</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="hBoxLayoutBottom" >
     <item>
      <widget class="QProgressBar" name="progressBar" >
       <property name="value" >
        <number>0</number>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hBoxLayoutDialogButton" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="buttonExport" >
         <property name="text" >
          <string>Convert</string>
         </property>
         <property name="shortcut" >
          <string/>
         </property>
         <property name="autoDefault" >
          <bool>true</bool>
         </property>
         <property name="default" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonClose" >
         <property name="text" >
          <string>&amp;Close</string>
         </property>
         <property name="shortcut" >
          <string>Alt+C</string>
         </property>
         <property name="autoDefault" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="gex.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonExport</sender>
   <signal>clicked()</signal>
   <receiver>ImportSpektraDialog_base</receiver>
   <slot>OnButtonExport()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonClose</sender>
   <signal>clicked()</signal>
   <receiver>ImportSpektraDialog_base</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSelect</sender>
   <signal>clicked()</signal>
   <receiver>ImportSpektraDialog_base</receiver>
   <slot>OnButtonSelect()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonProperties</sender>
   <signal>clicked()</signal>
   <receiver>ImportSpektraDialog_base</receiver>
   <slot>OnButtonProperties()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ImportSpektraDialog_base</sender>
   <signal>sSpektraFileSelected(QString)</signal>
   <receiver>editFileName</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ImportSpektraDialog_base</sender>
   <signal>sSpektraFileSelected(QString)</signal>
   <receiver>ImportSpektraDialog_base</receiver>
   <slot>OnSpektraFileSelected()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSetAllRecords</sender>
   <signal>clicked()</signal>
   <receiver>ImportSpektraDialog_base</receiver>
   <slot>OnButtonSetAllRecords()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonClearAllRecords</sender>
   <signal>clicked()</signal>
   <receiver>ImportSpektraDialog_base</receiver>
   <slot>OnButtonClearAllRecords()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboOutputFormat</sender>
   <signal>activated(int)</signal>
   <receiver>ImportSpektraDialog_base</receiver>
   <slot>OnOutputFormatChanged(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
